### PR TITLE
[4.2] Update Some Docblocks

### PR DESCRIPTION
**Pull request by RomainLanz**

### CODE REVIEW

File: src/Illuminate/Database/Schema/Blueprint.php [hunk 1]
```diff
@@ -661,6 +661,7 @@ public function binary($column)
 	 * Add the proper columns for a polymorphic table.
 	 *
 	 * @param  string  $name
+	 * @param  string  $indexName
```
Comment (GrahamCampbell) [inline src/Illuminate/Database/Schema/Blueprint.php]: Not correct. Should be `string|null`.

File: src/Illuminate/Html/FormBuilder.php [hunk 2]
```diff
@@ -392,6 +392,7 @@ protected function setQuickTextAreaSize($options)
 	 * Create a number input field.
 	 *
 	 * @param  string  $name
+	 * @param  string  $value
```
Comment (GrahamCampbell) [inline src/Illuminate/Html/FormBuilder.php]: Not correct. Should be `string|null`.

**Comment by RomainLanz** [comment]
Updated!

File: src/Illuminate/Html/FormBuilder.php [hunk 3]
```diff
@@ -392,6 +392,7 @@ protected function setQuickTextAreaSize($options)
 	 * Create a number input field.
 	 *
 	 * @param  string  $name
+	 * @param  string|null  $value
 	 * @param  array   $options
```
Comment (GrahamCampbell) [inline src/Illuminate/Html/FormBuilder.php]: please un-align this since you've added that other param

**Comment by RomainLanz** [comment]
Miss that, all done.

**Comment by GrahamCampbell** [comment]
:+1:

**Pull request merged**